### PR TITLE
feat(messagebox): add warning msgbox, add wc info in portal

### PR DESCRIPTION
**Pull request by lfbergee**
affects: @fremtind/jkl-message-box-react, @fremtind/jkl-message-box, @fremtind/portal

## 📥 Proposed changes

- add warning level to messagebox
- add fullwidth style for messagebox with lots of content
- add documentation for it
- add web components polyfill info to portal

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/message-box/message-box.scss [hunk 1]
```diff
@@ -8,6 +8,10 @@
     padding: $component-spacing--xl;
     background-color: $snøhvit;
 
+    &--full {
```
Comment (piofinn) [inline packages/message-box/message-box.scss]: Maybe `&--full-width` to make it even clearer?

File: packages/message-box-react/example/index.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom";
 import { InfoMessage, ErrorMessage, SuccessMessage } from "../src";
 import "@fremtind/jkl-message-box/message-box.scss";
 import "@fremtind/jkl-core/normalize.scss";
+import { WarningMessage } from "../src/MessageBox";
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply packages/message-box-react/example/index.tsx]: Yes, will fix

File: packages/message-box-react/src/MessageBox.tsx [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
     title: string;
     children: ReactNode;
+    fullWidth?: boolean;
+    className?: string;
 }
 
-type messageTypes = "info" | "error" | "success";
+type messageTypes = "info" | "error" | "success" | "warning";
+
+const css = "jkl-message-box";
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply packages/message-box-react/src/MessageBox.tsx]: I'm not sure either, about the name or the idea. Just wanted to try. BaseClass i is better.

File: packages/message-box/message-box.scss (hunk 1 above)
Comment (lfbergee) [reply packages/message-box/message-box.scss]: agreed

**Pull request merged**